2200086/10804.py

```diff
@@ -0,0 +1,20 @@
+# 문자열의 거울상
+
+T = int(input())
+for i in range(1, T+1):
+    a = input()
+    li = []
+    li2 = []    
+    for j in a:
+        if j == 'b':
+            li.append('d')
+        elif j == 'd':
+            li.append('b')
+        elif j == 'p':
+            li.append('q')
+        elif j == 'q':
+            li.append('p')
+    for m in li[::-1]:
+        li2.append(m)
```

Comment:
좋은 풀이네요! 👍 
```suggestion
    li2 = li[::-1]:
```
위와 같은 방법으로도 작성해볼 수 있을 것 같습니다 😄